app/models/user.rb

```diff
@@ -15,8 +15,7 @@ class User < ApplicationRecord
   validates :first_name, :last_name, :spire, :email, :phone, :rosters, presence: true
   validates :spire, :email, :phone, uniqueness: { case_sensitive: false }
   validates :calendar_access_token, uniqueness: { case_sensitive: true }
-  validates :spire, format: { with: /\A\d{8}@umass.edu\z/,
-                              message: 'must be 8 digits followed by @umass.edu' }
+  validates :spire, format: { with: /\A\d{8}@umass.edu\z/ }
```

Comment:
Also another preference I've built up from doing this sort of thing a lot on the hub - it might be a little nicer in terms of future maintenance to do something like

```suggestion
  validates :spire, format: { with: /\A\d{8}@umass.edu\z/, message: :must_be_eight_digits_and_umass }
```

combined with

```yaml
en:
  ...
  errors:
    messages:
      must_be_eight_digits_and_umass: 'must be 8 digits followed by @umass.edu'
```

i.e. the explicit key here makes it easier to infer that this is overridden with a custom translation, as well as makes it much easier to find said translation (especially when the translation file gets bigger)